src/main/resources/addon.yml

```diff
@@ -325,4 +328,7 @@ permissions:
     default: true
   parkour.parkour.removewarp:
     description: Allow use of '/parkour removewarp' command - removes the warp spot and makes your course private
-    default: true   
\ No newline at end of file
+    default: true
+  parkour.parkour.quit:
```

Comment:
If all there permissions had parkour. Removed from them, then this needs to be updated as well, as well as a strong warning as it might break existing permissions setups